bm-persona/Fitness/FitnessViewController.swift

```diff
@@ -212,19 +271,40 @@ extension FitnessViewController {
         card.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
         card.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
         card.heightAnchor.constraint(equalTo: view.layoutMarginsGuide.heightAnchor).isActive = true
+        gymCard = card
         
-        let table = UITableView()
-        table.delegate = self
-        table.dataSource = self
-        scrollView.addSubview(table)
-        table.translatesAutoresizingMaskIntoConstraints = false
-        table.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
-        table.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor).isActive = true
-        table.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
-        table.bottomAnchor.constraint(equalTo: card.layoutMarginsGuide.bottomAnchor).isActive = true
+        let fitnessLabel = UILabel()
+        fitnessLabel.text = "Fitness Centers"
+        fitnessLabel.font = kHeaderFont
+        fitnessLabel.adjustsFontSizeToFitWidth = true
+        fitnessLabel.textColor = .black
+        fitnessLabel.textColor = Color.blackText
+        card.addSubview(fitnessLabel)
         
-        gymCard = card
-        gymsTable = table
+        setupFilterTableView()
+        card.addSubview(filterTableView)
+        
+        fitnessLabel.translatesAutoresizingMaskIntoConstraints = false
+        fitnessLabel.heightAnchor.constraint(equalToConstant: 30).isActive = true
+        fitnessLabel.leftAnchor.constraint(equalTo: card.layoutMarginsGuide.leftAnchor, constant: 8).isActive = true
+        fitnessLabel.rightAnchor.constraint(equalTo: card.layoutMarginsGuide.rightAnchor).isActive = true
+        fitnessLabel.topAnchor.constraint(equalTo: card.layoutMarginsGuide.topAnchor).isActive = true
+        
+        filterTableView.translatesAutoresizingMaskIntoConstraints = false
+        filterTableView.topAnchor.constraint(equalTo: fitnessLabel.layoutMarginsGuide.bottomAnchor, constant: 14).isActive = true
```

Comment:
Let's keep the label's left anchor to be equal to the card's layout margins' left anchor, and let's keep the vertical spacing between the label and the tableview equal to kViewMargin. I know it's different in the mockup, but it's more consistent with the rest of this ViewController and I think it's probably what they intended, haha.

Reply:
The reason the left anchor was adjusted by 8 was because the FilterTableView had an automatic left inset of 8, so I wanted to line up the label and table. Instead, I've modified the FilterTableView's layoutMargins to not be inset by 8.